recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "votca" %}
-{% set version = "2022.0" %}
+{% set version = "2022" %}
```

Comment:
Or should we keep the last digit?

Reply:
Your call, I just wanted to make it consistent with upstream.

Reply:
Ok, let's give it a try